### DISCUSSION TITLE
How to invert a character in format

### DISCUSSION THREAD

**Discussion by Acisama** [starship/starship#6433] [Q&A]
In my format, I want to have the username followed by a right facing triangle and then a bit of space and then an inverted triangle (kinda like this https://github.com/catppuccin/windows-terminal).

But I just can't get that inverted triangle, it always defaults the color of fg to grey (and black doesn't work because my background is transparent).
I also tried the unicode character 🭨  but it doesn't have quite the right shape.

Could someone give an example line of code of how to do this?

Thanks

**Comment by anuanandpremji** [comment]
Also stuck on the same problem. Using `\ue0d7` (🭨) was the only solution I found as well. 

To get it to work, I had to change my nerd font through trial and error to display it in a way that didn't seem jarring. Would love to have a proper solution instead.

Basically, as I understand, there is no way currently for starship prompt to do a prompt similar to https://github.com/romkatv/agnoster-gaps, without hard-coding format styles to match terminal background. The problem with that is if one switches the system theme from dark theme to light theme and the terminal is configured to follow, system theme, the prompt looks broken.

Ideally, there should be a way to either invert a character or to set `fg:none` and `bg:#color`for a style.
